docs/es/getting_started/update.md

```diff
@@ -58,11 +58,11 @@ B. Sólo actualizar hasta cierta versión (en el caso de que prefieras actualiza
 
```

Comment:
Again, not related to this line :sweat_smile:.

A few lines above this one, it says "si estas"; should be "si estás".

And another thing on that line (also in the English version). Since we stopped adding `v` at the beginning of tags, how about using `1.3.0` or `1.3.1` as an example, since they follow the new tag format convention?